src/screens/OrganizationEvents/OrganizationEvents.tsx

```
@@ -206,28 +370,44 @@
+        }
+
         const input = {
           name: formState.name,
           description: formState.eventdescrip,
           startAt: alldaychecked
-            ? dayjs(startDate).startOf('day').toISOString()
+            ? dayjs(startDate)
+                .startOf('day')
+                .format('YYYY-MM-DDTHH:mm:ss.SSS[Z]')
             : dayjs(startDate)
                 .hour(parseInt(startTimeParts[0]))
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)